Source/AssetRipper.Processing/AnimationClips/AnimationClipConverter.cs

```diff
@@ -40,7 +39,7 @@ private AnimationClipConverter(IAnimationClip clip, PathChecksumCache checksumCa
 
 		public static void Process(IAnimationClip clip, AnimationCache cache, PathChecksumCache checksumCache)
 		{
-			AnimationClipConverter converter = new AnimationClipConverter(clip, checksumCache);
+			AnimationClipConverter converter = new(clip, checksumCache);
```

Comment:
I'm only going to comment on this line, but the problem is pervasive in this pr. Formatting changes have no place in pull requests. They make git blaming more annoying and distract pull request reviewers. If I want to make formatting changes, I'll use an auto-formatter and do the whole project en masse in a single commit.